package-lock.json

```diff
@@ -7077,20 +7350,279 @@
       "integrity": "sha512-u1wmTI1jJGzCJzWndZo8mk4wnPTZd1eOIYTYvuEyOQGfmDl3TrabCCfKnOC86FZwW/9djqTl933UF/cS425i9A=="
     },
     "node_modules/cuix": {
-      "version": "13.0.4",
-      "resolved": "file:desktop/core/src/desktop/static/desktop/ext/cuix/cuix-13.0.4.tgz",
-      "integrity": "sha512-q2cJTt0DjDl0TEjxKqY0BmiK6fEwWNMcJ7J8CFa6kYw6DQKCItSuZHr7Mxivb05NAKOLkr/OE77f1GTC8eCTDg==",
+      "version": "13.4.0",
+      "resolved": "file:desktop/core/src/desktop/static/desktop/ext/cuix/cuix-13.4.0.tgz",
+      "integrity": "sha512-fPS5zregNspNL5ElGGpAgi/9TIp2Iond8zVeAYxehbr03aj2n0KYSvOJBl+ugKGcHdUjXp5+CRRJyzTo5tNlsg==",
       "dependencies": {
+        "@dnd-kit/core": "6.1.0",
+        "@dnd-kit/modifiers": "7.0.0",
+        "@dnd-kit/sortable": "8.0.0",
         "@fullstory/browser": "1.6.2",
+        "@react-hookz/web": "^23.1.0",
+        "compare-versions": "5.0.3",
+        "dompurify": "2.0.11",
         "javascript-autocomplete": "1.0.5",
         "js-cookie": "2.2.1",
+        "moment-timezone": "0.5.31",
         "popper.js": "1.16.0",
-        "preact": "10.11.0"
+        "preact": "10.11.0",
+        "react-redux": "8.0.5",
+        "recharts": "2.1.9",
+        "redux": "4.0.5",
+        "redux-saga": "1.1.3",
+        "styled-components": "6.1.8",
+        "styled-theming": "2.2.0"
```

Comment:
@bjornalm these new dependencies seems to break Hue, when you did the initial include of cuix did you also remove these somehow?